cartography/data/jobs/analysis/aws_eks_asset_exposure.json

```diff
@@ -8,7 +8,7 @@
 		},
 		{
 			"__comment": "This sets the exposed_internet attribute",
-			"query": "MATCH (cluster:EKSCluster)<-[:RESOURCE]-(:AWSAccount{id: $AWS_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE cluster.endpoint_public_access = true AND '0.0.0.0/0' IN cluster.public_access_cidrs SET cluster.exposed_internet = true,(CASE WHEN NOT 'direct' IN coalesce(cluster.exposed_internet_type , []) THEN cluster END).exposed_internet_type = coalesce(cluster.exposed_internet_type , []) + 'direct';",
+			"query": "MATCH (cluster:EKSCluster)<-[:RESOURCE]-(:AWSAccount{id: $AWS_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE cluster.endpoint_public_access = true AND '0.0.0.0/0' IN cluster.public_access_cidrs OR '::/0' IN cluster.public_access_cidrs SET cluster.exposed_internet = true,(CASE WHEN NOT 'direct' IN coalesce(cluster.exposed_internet_type , []) THEN cluster END).exposed_internet_type = coalesce(cluster.exposed_internet_type , []) + 'direct';",
```

Comment:
add brackets around the or condition. Otherwise, even though endpoint_public_access is false, if `'::/0' IN cluster.public_access_cidrs` condition would be true, the entire condition will be true.
new condition should be:
```
cluster.endpoint_public_access = true AND ('0.0.0.0/0' IN cluster.public_access_cidrs OR '::/0' IN cluster.public_access_cidrs)
```